doc: add Hebrew tutorial

TzviGreenfeld opened the pull request:
Participates in #267 
Let me know if you have any feedback, thanks.

Lucas-C commented on docs/Tutorial-he.md:
```diff
@@ -0,0 +1,162 @@
+
+על מנת ליצור קישור חיצני באמצעות תמונה, השתמשנו במתודה [()image](https://pyfpdf.github.io/fpdf2/fpdf/fpdf.html#fpdf.fpdf.FPDF.image). למתודה יש אופציה לקבל קישור כאחד הפרמטרים שלה. הקישור יכול להיות פנימי או חיצוני.
+
+ניתן גם להשתמש במתודה `()write` כדי לשנות סגנונות גופן ולהוסיף קישורים. זהו פארסר של html, שמאפשר להוסיף טקסט, לשנות את הסגנון ולהוסיף קישורים באמצעות html.
```
```suggestion
ניתן גם להשתמש במתודה `()write_html` כדי לשנות סגנונות גופן ולהוסיף קישורים. זהו פארסר של html, שמאפשר להוסיף טקסט, לשנות את הסגנון ולהוסיף קישורים באמצעות html.
```

Lucas-C commented:
@TzviGreenfeld thank you for submitting this!

Could you also add a mention of this new tutorial in `CHANGELOG.md`, `docs/index.md`, `tutorial/notebook.ipynb`, `mkdocs.yml` & `README.md` please?
Take instpiration on the previous tutorial translations 😊

TzviGreenfeld commented:
updated those files. Im just not really sure how is the html generated from md. anyway I've added links to docs/Tutorial-he.html (this file doesn't exist yet)

Lucas-C commented:
That's perfect, thank you @TzviGreenfeld!

We generate the HTML version of our documentation using [mkdocs](https://www.mkdocs.org)

Merging this PR now

@all-contributors please add @TzviGreenfeld for documentation

Lucas-C commented:
Aaaaaand it's live! https://pyfpdf.github.io/fpdf2/Tutorial-he.html

Lucas-C commented:
I have asked the quetion here: https://github.com/squidfunk/mkdocs-material/discussions/4465

Lucas-C commented:
For reference, using the [markdown-in-html](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html) Markdown extension did not solve the issue

Lucas-C commented:
Solved thanks to @facelessuser help here:
https://github.com/squidfunk/mkdocs-material/discussions/4465#discussioncomment-3816820
